docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/qualys-vmdr-source.md

```diff
@@ -50,14 +50,17 @@ To configure a Qualys VMDR Source:
 6. (Optional) **Fields**. Click the **+Add Field** link to define the fields you want to associate. Each field needs a name (key) and value.
    * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
-   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+	 * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
 7. **Qualys API Server URL** and **Qualys API Gateway URL**. Provide the Qualys API server URLs. Use the [Qualys Platform Identification](https://www.qualys.com/platform-identification) page and scroll down to **API URLs** to for a reference to your Qualys deployment location.
 8. **Username** and **Password**. Use your Qualys account username and password for API authentication.
 9. The next section covers the type of data to collect and how often. <br/> <img src={useBaseUrl('img/send-data/qualys-config-collection-selection.png')} alt="qualys-config-collection-selection.png" width="450" />
 10. **Collect vulnerability data**. This option will fetch the list of hosts with the host's latest vulnerability data based on the host-based scan data available in the user’s account. We recommend leaving the polling interval at the default 1 hour.
 11. (Optional) **Collect KnowledgeBase Information**. This option is only available if you choose to collect vulnerability data. If selected, it will automatically download the vulnerability details from the Qualys KnowledgeBase for vulnerabilities detected within your environment.
 12. **Collect asset inventory**. This option consumes asset data from Qualys Global IT Asset Inventory API. The inventory data collected here will also be used in Cloud SIEM as inventory data. We recommend leaving the polling interval at the default 24 hours.
-13. When you are finished configuring the Source, click **Submit**.
+	 * To forward Qualys VMDR assetInventory to CSE, it is recommended to create a [Field Extraction Rule](https://help.sumologic.com/docs/manage/field-extractions/create-field-extraction-rule/) with the following criteria:
+		 * Scope: _sourceCategory="Enter Your Source Category Here from Step 5" "assetInventory"
+	   * Parse Expression: "true" as _siemForward
```

Comment:
@mvirga-sumo , I've applied the code font where it looks like it's needed. Can you please take a look?

```suggestion
   * To forward Qualys VMDR assetInventory to CSE, it is recommended to create a [Field Extraction Rule](https://help.sumologic.com/docs/manage/field-extractions/create-field-extraction-rule/) with the following criteria:
      * Scope: `_sourceCategory="<enter your source category here from Step 5>" "assetInventory"`
      * Parse Expression: `true` as `_siemForward`
```

Reply:
@kimsauce Thank you for reviewing the PR. The double quotes on line 62 are required. If it's possible to put _sourceCategory="Enter Your Source Category Here from Step 5" "assetInventory" in a code block as well as "true" as _siemForward in a code block that would be excellent! 

Thank you!

Reply:
@mvirga-sumo like this?

* Scope:
    ```
    _sourceCategory="<enter your source category here from Step 5>" "assetInventory"
   ```
* Parse Expression:
   ```
   "true" as _siemForward
   ```

Reply:
@kimsauce Yep that is perfect. Thank you!